api/v1beta1/openstackprovisionserver_types.go

```diff
@@ -35,6 +35,8 @@ const (
 	ProvisioningNetworkDisabled  ProvisioningNetwork = "Disabled"
 	// Checksum job hash
 	ChecksumHash = "checksum"
+	// DefaultProvisionPort - The starting default port for the OpenStackProvisionServer's Apache container
+	DefaultProvisionPort = 6190
```

Comment:
Should we reserve a port range and document that to avoid potential conflicts on OCP nodes?

Reply:
We  increment and auto-assign ports from 6190 (with no upper limit) and user can set any port they want in the spec. I was talking about validating that all the provision server ports (either auto-assigned or user provided) are within a range. That could be be a separate PR.